app/views/report/_column_lists.html.haml

```diff
@@ -16,20 +16,28 @@
       %td
     %tr
       %td{:align => "center"}
```

Comment:
@romanblanco try this instead of: %td{:align => "center"}

%td.text-center
  .btn-group-vertical